modules/shell/example_test.go

```diff
@@ -0,0 +1,35 @@
+package shell_test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/shell"
+)
+
+func ExampleCommand() {
+	var t *testing.T
+
+	out, err := shell.RunCommandAndGetOutputE(t, shell.Command{
+		Command: "sh",
+		Args:    []string{"-c", `echo "hello world" && echo "this command has failed" >&2 && exit 1`},
+	})
+	t.Logf("Failure.\n\tOutput: %v\n\tError: %v", out, err)
```

Comment:
Will this also print `stderr`? Even without the casting below?

Reply:
Yes; the error.String now always contains `stderr`. Output too, depending on the method that is called (`shell.RunCommandAndGetOutput` gets both stdout & stderr)

Reply:
Nice!